sphinx_redirect_by_id/redirecttools.js

```diff
@@ -26,10 +26,8 @@ var Redirect = {
       const query = new URLSearchParams(window.location.search).get("q");
       if(query) {
           Redirect.performSearch(query);
-      } else {
-          const out = document.getElementById('search-results');
-          out.innerHTML = '<p>No query ID provided. Use <code>?q=theId</code> to redirect.</p>';
-      }
+      } else
+        Redirect.redirectToSearch('');
```

Comment:
I don't think it should redirect here. If the user didn't provide ``q=<something>`` then they should be told how to use the redirection interface.

Reply:
All right, makes sense.